default.py

```diff
@@ -1207,8 +1195,7 @@ def get_list(listname, datadir=__datapath__):
     @staticmethod
     def load_file(file_path):
         try:
-            with open(file_path, 'r') as f:
-                return eval(f.read())
+            return eval(file(file_path, "r").read())
```

Comment:
What format is this file? Using eval should be avoided as much as possible as it can be turned easily into a security hole. Better to use JSON, elementtree, etc depending on the file.

Reply:
I agree, there are quite a bit of things that are legacy in this script (repr/eval), it was created back before json was even included with python.

the file format is a python dict